rtcclient/client.py

```diff
@@ -32,6 +32,11 @@ class RTCClient(RTCBase):
         the url ends with 'jazz', otherwise to `False` if with 'ccm'
         (Refer to issue #68 for details)
     :type ends_with_jazz: bool
+    :param attribute_map: (optional) Dictionary mapping RTC attributes to
```

Comment:
Actually I don't get what this `attribute_map` is for.

If you want to know all the alias, you can refer to `field_alias`. See [this link](https://rtcclient.readthedocs.io/en/latest/quickstart.html#about-returned-properties).

Reply:
`attribute_map` does two things:

1. It allows overriding the alias that is automatically created in `FieldBase.__initializeFromRaw`.
2. It is the mechanism for forcing the inclusion of an attribute with a "." in the name, which is the specific fix for issue #108

Reply:
Hello.  Just following up.  Is there anything else I can do to move this along?

Reply:
Can `field_alias` in this [link](https://rtcclient.readthedocs.io/en/latest/quickstart.html#about-returned-properties) be used, instead of having a new `attribute_map` ?